dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Abstractions/IGateway.cs

```diff
@@ -1,18 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // IGateway.cs
+using Grpc.Core;
 using Microsoft.AutoGen.Contracts;
+using Microsoft.AutoGen.Protobuf;
 
-namespace Microsoft.AutoGen.Runtime.Grpc.Abstractions;
+namespace Microsoft.AutoGen.RuntimeGateway.Grpc.Abstractions;
 
 public interface IGateway : IGrainObserver
 {
     ValueTask<RpcResponse> InvokeRequestAsync(RpcRequest request);
     ValueTask BroadcastEventAsync(CloudEvent evt);
-    ValueTask StoreAsync(Contracts.AgentState value);
-    ValueTask<Contracts.AgentState> ReadAsync(AgentId agentId);
-    ValueTask<RegisterAgentTypeResponse> RegisterAgentTypeAsync(RegisterAgentTypeRequest request);
+    ValueTask<RegisterAgentTypeResponse> RegisterAgentTypeAsync(RegisterAgentTypeRequest request, ServerCallContext context);
     ValueTask<AddSubscriptionResponse> SubscribeAsync(AddSubscriptionRequest request);
     ValueTask<RemoveSubscriptionResponse> UnsubscribeAsync(RemoveSubscriptionRequest request);
     ValueTask<List<Subscription>> GetSubscriptionsAsync(GetSubscriptionsRequest request);
-    Task SendMessageAsync(IConnection connection, CloudEvent cloudEvent);
+    Task WriteResponseAsync(IConnection connection, CloudEvent cloudEvent);
```

Comment:
What does this mean? A cloud event is never a response

Reply:
It's written to the response stream from the gateways point of view. before it was called send message and I renamed it to avoid confusion. clearly unsuccessfully. Suggestions?

Reply:
I see, I was mainly confused why there are two different cloud event related methods

Reply:
it doesn't need to be public any longer, removing